R/reporter-progress.R

```diff
@@ -286,6 +291,10 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
       }
       self$last_update <- time
       TRUE
+    },
+
+    get_verbose_skips = function() {
```

Comment:
To keep this as simple as possible, lets drop the accessor function

Reply:
@wch: Can I access public member variables from a subclass? It doesn't appear so, in this file in `ParallelProgressReporter$add_result()` I only see public functions from `super$`, and https://r6.r-lib.org/articles/Introduction.html#inheritance-1 doesn't talk about accessing fields from subclasses.

Reply:
@krlmlr You should be able to access a public field from a subclass.